rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SessionStore.java

```diff
@@ -120,4 +145,30 @@ public void renewSession(String sessionId) throws NotConnectedException {
         }
     }
 
+    public void renewIAMSession(String sessionId) throws NotConnectedException {
+
+        IAMSession iamSession = (IAMSession) get(sessionId);
```

Comment:
I don't understand this mechanism

1) you renew only the local rest server session
2) you store the user credentials in the jwt token (very insecure)
3) you recreate from scratch new iam sessions, which lead to renewIAMSession(somesessionid) will lead to a creation of another sessionid without the user knowing its sessionid changed. How is that supposed to work ?

There MUST be a way to call the IAM and ask it to renew a session, by returning an eventual new token lease.